src/js/lib/utils.js

```diff
@@ -391,14 +391,8 @@ var utils = {
 
     getSystemFonts: function () {
         var defaultFontDirs = {
-            win32: function () {
-                if (process.env.LOCALAPPDATA != undefined) {
-                    return ['/Windows/Fonts', process.env.LOCALAPPDATA + '/Microsoft/Windows/Fonts'];
-                } else {
-                    return ['/Windows/Fonts'];
-                }
-            },
-            darwin: () => ['/Library/Fonts'],
+            win32: () => ['/Windows/Fonts', os.homedir() + '/AppData/Local/Microsoft/Windows/Fonts'],
```

Comment:
**LOCALAPPDATA** : C:/Users/{username}/AppData/Local (see env var [docs](https://ss64.com/nt/syntax-variables.html))
[os.homedir()](https://nodejs.org/docs/latest-v6.x/api/os.html#os_os_homedir) : C:/Users/{username}

Using os.homedir() is cleaner and unified.